submit_cgap/submission.py

```diff
@@ -777,25 +798,23 @@ def execute_prearranged_upload(path, upload_credentials, auth=None):
     try:
         source = path
         target = upload_credentials['upload_url']
-        show("Going to upload %s to %s." % (source, target))
+        show("Uploading local file %s directly (via aws-cli) to: %s" % (source, target))
```

Comment:
The hyphen in `aws-cli` is troubling to me because it seems to refer to a specific named tool. The named library is `awscli` and the command is `aws` and you're using a third name that just promotes confusion. I think you could say `aws CLI` (with a space between command name and juxtaposed acronym, and you'd have to change the corresponding test. I'd also be OK with `AWS CLI`. Mostly I just want a space, not a hyphen.